assets/schema/ensemble_schema.json

```diff
@@ -1206,6 +1206,19 @@
                 "color": {
                   "$ref": "#/$defs/type-color",
                   "description": "Set the color for the QR code drawing"
+                },
```

Comment:
can you please add the scheme into studio as well. the schema needs to be add in ts file check studio repo readme.